tests/storage/storage.go

```diff
@@ -1222,7 +1236,17 @@ var _ = SIGDescribe("Storage", func() {
 				vmi1, vmi2 *virtv1.VirtualMachineInstance
 			)
 			BeforeEach(func() {
-				dv = libstorage.NewDataVolumeWithRegistryImport(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+				sc, exists := libstorage.GetRWOFileSystemStorageClass()
+				if !exists {
+					Skip("Skip test when Filesystem storage is not present")
+				}
+
+				dv = libdv.NewDataVolume(
+					libdv.WithNamespace(util.NamespaceTestDefault),
+					libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros)),
+					libdv.WithPVC(sc, libdv.PVCSizeForRegistryImport, k8sv1.ReadWriteOnce, k8sv1.PersistentVolumeFilesystem),
```

Comment:
I would leave this `PVCSizeForRegistryImport` as a const that is either defined in this test, or some other place and name it as CirrosPVCSize, because it does not have anything in common with `RegistryImport` and nothing with `libdv` 

I really like this PR and I would it to finally be merged (before another merge conflict). So I think we can discuss and  decide if this detail has to be done now, or later.